index.js

```diff
@@ -0,0 +1,55 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var mongoose = require('mongoose');
+var expressJWT = require('express-jwt');
+var jwt = require('jsonwebtoken');
+var Bike = require("./models/bike");
+var User = require("./models/user");
+var app = express();
+
+var secret = "this_is_my_secret_thing"
+
+mongoose.connect('mongodb://localhost:27017/mybikes');
+
+app.use("/api/users", expressJWT(
+	{
+		secret: secret
+	}
+).unless({ method: "POST" }));
+
+app.use(bodyParser.urlencoded({extended:true}));
+// app.use('/api/bikes', require('./controllers/bikes'));
```

Comment:
Not sure why you commented this out. I had to uncomment it out in order to use postman for bikes, but your routes seemed to work alright.